jackson/src/main/resources/reference.conf

```diff
@@ -1,40 +1,25 @@
 play.modules.enabled += com.lightbend.lagom.javadsl.jackson.JacksonModule
 
-lagom.serialization.json {
-  
-  #//#jackson-modules
-  # The Jackson JSON serializer will register these modules.
-  # It is also possible to use jackson-modules = ["*"] to dynamically
-  # find and register all modules in the classpath.  
-  jackson-modules += "com.fasterxml.jackson.module.paramnames.ParameterNamesModule"
-  jackson-modules += "com.fasterxml.jackson.datatype.jdk8.Jdk8Module"
-  jackson-modules += "com.fasterxml.jackson.datatype.jsr310.JavaTimeModule"
-  jackson-modules += "com.fasterxml.jackson.datatype.pcollections.PCollectionsModule"
-  jackson-modules += "com.fasterxml.jackson.datatype.guava.GuavaModule"
-  #//#jackson-modules  
-  
-  # The serializer will compress the payload if the message class
-  # implements the CompressedJsonable marker interface and the payload 
-  # is larger than this value. Only used for remote messages within 
-  # the cluster of the service.
-  compress-larger-than = 1024b
-  
-  # Define data migration transformations of old formats to current 
-  # format here as a mapping between the (old) class name to be
-  # transformed to the JacksonJsonMigration class that implements
-  # the transformation.  
-  migrations {
-  }
-}
-
 akka.actor {
   serializers {
-    lagom-json = "com.lightbend.lagom.internal.jackson.JacksonJsonSerializer"
+    # Placeholder for Lagom 1.5.x JacksonSerializer.
+    # Needed for deserialization of messages during rolling update, and old data (events, snapshots)
+    # that were serialized with the Lagom 1.5.x JacksonSerializer.
+    # Note that by default there is no
+    old-lagom-json = "com.lightbend.lagom.internal.jackson.OldJacksonJsonSerializer"
   }
   serialization-bindings {
-    "com.lightbend.lagom.serialization.Jsonable" = lagom-json
+    # Bind Jsonable to Akka JacksonJsonSerializer
+    "com.lightbend.lagom.serialization.Jsonable" = jackson-json
```

Comment:
I initially thought that we should default to `old-lagom-json` to allow rolling updates in two phases. 

On the other hand, since we will be defaulting to Artery we will need a cluster shutting down. Therefore, in the migration guide, when we explain how to avoid a cluster shutting down, we should explain how to use the old serializer to avoid disruption. 

WDYT?

Reply:
I think the artery case is slightly different because it was available in the past, but not the default. this is a completely new feature.

In any case, I think I prefer defaults to be forward-looking. This way things crash early and users are required to think of the migration or opt-in to classic impl.

Reply:
I agree. Better to have the default forward-looking, and rolling update is a single occurrence special case that requires some adjustments.

Reply:
I have removed the OldJacksonSerializer class. Wasn't needed any more since the Akka JacksonSerializer can be bound for deserialization to the old serializerId. See df84c104b5679f870cf79a747b9086f2c84b5dd0

Moved the migration config check to the JacksonModule.

Are you ok with this "source incompatibility" or do we have to find a way to bridge old and new `JacksonJsonMigration` classes and config?

Reply:
I'm fine with it.